ms-inventory/routes/controller.go

```diff
@@ -0,0 +1,85 @@
+// Copyright © 2022 Intel Corporation. All rights reserved.
+// SPDX-License-Identifier: BSD-3-Clause
+
+package routes
+
+import (
+	"fmt"
+
+	"github.com/edgexfoundry/app-functions-sdk-go/v2/pkg/interfaces"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/clients/logger"
+)
+
+type Controller struct {
+	lc             logger.LoggingClient
+	service        interfaces.ApplicationService
+	inventoryItems Products
+	auditLog       AuditLog
+}
+
+func NewController(lc logger.LoggingClient, service interfaces.ApplicationService) Controller {
+	return Controller{
+		lc:      lc,
+		service: service,
+	}
+}
+
+func (c *Controller) AddAllRoutes() error {
+	var err error
+
+	err = c.service.AddRoute("/inventory", c.InventoryGet, "GET")
```

Comment:
use pre-defined constant:
```suggestion
	err = c.service.AddRoute("/inventory", c.InventoryGet, http.MethodGet)
```